activesupport/test/file_evented_update_checker_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'abstract_unit'
+require 'fileutils'
+require 'thread'
+require 'file_update_checker_with_enumerable_test_cases'
+
+MTIME_FIXTURES_PATH = File.expand_path("../fixtures", __FILE__)
+
+class FileEventedUpdateCheckerWithEnumerableTest < ActiveSupport::TestCase
+	include FileUpdateCheckerWithEnumerableTestCases
+  def build_new_watcher(files, dirs={}, &block)
```

Comment:
Please add a blank line before the method definition.

Also please add a space before and after the =``.